keepassxc-browser/background/browserAction.js

```diff
@@ -52,8 +52,13 @@ browserAction.generateIconName = function(iconType) {
     let name = 'icon_';
     name += (keepass.keePassXCUpdateAvailable()) ? 'new_' : '';
     name += (!iconType || iconType === 'normal') ? 'normal' : iconType;
-
-    return `/icons/toolbar/${name}.png`;
+    
+    let style = 'colored'
+    if (page.settings.useMonochromeToolbarIcon) {
+        style = window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
```

Comment:
This seem to only apply to the System theme now, right? If I manually switch to Light theme in the extension settings, the dark icons will be still used.

Reply:
Good catch! I haven't thought of the theme setting...

Reply:
Fixed in [86bdfe4](https://github.com/keepassxreboot/keepassxc-browser/pull/1938/commits/86bdfe441368fd0838cd3b5f499f0c7eb19fd7d0)